tools/packaging/common/istio-start.sh

```diff
@@ -21,7 +21,7 @@
 set -e
 
 # Match pilot/docker/Dockerfile.proxyv2
-export ISTIO_META_ISTIO_PROXY_VERSION="1.1.3"
+export ISTIO_META_ISTIO_VERSION="1.3.0"
```

Comment:
should this be hard coded?

Reply:
this is for some VM file.. I simply followed the existing pattern.

Reply:
I though the whole point of this was to remove the manual work. if we have to update this and the makefile anyways what did we really gain?

Reply:
This is another issue.